bahmni_account/models/account_invoice_line.py

```diff
@@ -4,39 +4,3 @@
 class AccountInvoiceLine(models.Model):
     _inherit = 'account.move.line'
     
-    @api.model
-    def create(self, vals):
```

Comment:
Is this not being used anywhere, and hence deleted?

Reply:
exactly